modular_skyrat/modules/customization/modules/jobs/_job.dm

```diff
@@ -117,25 +117,83 @@
 /datum/job/prisoner
 	required_languages = null
 
+// START OF BUBBERSTATION ADDITION
+
+/datum/job/mime
+	required_languages = null
+
+/datum/job/clown
+	required_languages = null
+
+/datum/job/bouncer
+	banned_quirks = list(GUARD_RESTRICTED_QUIRKS)
+
+/datum/job/barber
+	required_languages = null
+
+/datum/job/bartender
+	required_languages = null
+
+/datum/job/chaplain
+	required_languages = null
+
+// END OF BUBBERSTATION ADDITION
+
+// ENGINEERING (BUBBERSTATION ADDITION)
+
 /datum/job/station_engineer
 	banned_quirks = list(TECH_RESTRICTED_QUIRKS)
+	required_languages = null // BUBBERSTATION ADDITION
 
 /datum/job/atmospheric_technician
 	banned_quirks = list(TECH_RESTRICTED_QUIRKS)
+	required_languages = null // BUBBERSTATION ADDITION
 
-/datum/job/orderly
+/datum/job/engineering_guard
 	banned_quirks = list(GUARD_RESTRICTED_QUIRKS)
 
-/datum/job/science_guard
-	banned_quirks = list(GUARD_RESTRICTED_QUIRKS)
+// CARGO (BUBBERSTATION ADDITION)
+/datum/job/cargo_technician
+	required_languages = null
+
+/datum/job/shaft_miner
+	required_languages = null
+
+/datum/job/blacksmith
+	required_languages = null
 
 /datum/job/customs_agent
 	banned_quirks = list(GUARD_RESTRICTED_QUIRKS)
 
-/datum/job/bouncer
+// MEDICAL (BUBBERSTATION ADDITION)
+
+/datum/job/chemist
+	required_languages = null
+
+/datum/job/doctor
+	required_languages = null
+
+/datum/job/paramedic
+	required_languages = null
+
+/datum/job/geneticist
+	required_languages = null
+
+/datum/job/virologist
+	required_languages = null // damara dent-head
+
```

Comment:
This is a bad idea, in my opinion. A virologist cannot release diseases unapproved by the CMO - How are they going to get them approved if they can't talk to one another?

Reply:
Dunno ask psychotic moth chick